shotgun_api3/shotgun.py

```diff
@@ -345,6 +345,10 @@ def __init__(self, sg):
         """
         self._sg = sg
         self.max_rpc_attempts = 3
+        # The interval between _call_rpc retries.  Since the Config can't be
+        # modified until the Shotgun instance is created, use an environment
+        # variable to override the default value.
+        self.rpc_attempt_interval = os.environ.get("SHOTGUN_API_RETRY_INTERVAL", 3)
```

Comment:
Could this also be an optional argument when a `Shotgun` instance is initialized? I'm thinking it would work like the `ca_certs` argument, where it can be passed in OR set via an env var. You can store the retry sleep as a property on the `Shotgun` object and `_Config` can read it from there.

That has the added benefit of giving us a good place to document this and the environment variable.

Reply:
Also, however this is documented, it would be good to specifically mention that it's expecting to receive a value in seconds.

Reply:
I think there’s a bug with your env var. If it is set, you’ll get a string back, not an int. You’ll need to convert the value before using it 

Maybe I suggest having the evr var be in milliseconds instead of seconds? There sort of time settings are usually in milliseconds not seconds.

Reply:
I added a kwarg to Shotgun's __init__, and I'm now storing the value on the _Config instance.  I kept the rpc_attempt_interval property on _Config, since this seemed consistent with other similar properties (notably max_rpc_attempts), but I'm happy to move it to the Shotgun object if you think that would be cleaner.

Added a cast to int when getting the env var value, and am now storing wait in ms.